.github/workflows/build-linux.yml

```diff
@@ -18,16 +18,16 @@ jobs:
   build-linux:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: lukka/get-cmake@latest
-      - uses: actions/setup-java@v3
+      - uses: actions/setup-java@v4
         with:
-          distribution: temurin
+          distribution: oracle
```

Comment:
For now we are aligning on using `oracle` here and in the applet viewer

Reply:
Nevermind, doesn't support `x86` :: https://github.com/actions/setup-java/blob/main/src/distributions/oracle/installer.ts#L54-L56